bugbug/models/defect.py

```diff
@@ -243,7 +243,10 @@ def get_bugbug_labels(self, kind="bug"):
         return {bug_id: label for bug_id, label in classes.items() if bug_id in bug_ids}
 
     def get_labels(self):
-        return self.get_bugbug_labels("bug")
+        return (
+            self.get_bugbug_labels("bug"),
+            sorted(list(set(self.get_bugbug_labels("bug").values())), reverse=True),
```

Comment:
We have a predefined set of classes here, you can just return them directly (1 and 0).